glaemresources/modes/english-cirth-erebor.glaem

```diff
@@ -0,0 +1,222 @@
+\**
+
+Glǽmscribe (also written Glaemscribe) is a software dedicated to
+the transcription of texts between writing systems, and more 
+specifically dedicated to the transcription of J.R.R. Tolkien's 
+invented languages to some of his devised writing systems.
+
+Copyright (C) 2015 Benjamin Babut (Talagan).
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+**\
+
+\beg changelog
+  \entry "0.0.1", "Adapted to an English instead of Khuzdul mode from khuzdul-cirth-moria.glaem"
+\end
+
+\beg options
+  \option dots_for_spaces false
+\end
+
+\** This is the version of the Cirth used in the Title page of the Lord of the Rings **\
+\** See https://ring-lord.tripod.com/cirth/angerthaserebor.htm for a good reference **\
+\language "English"
+\writing  "Cirth"
+\mode     "English Cirth - Angerthas Erebor"
+\version  "0.0.1"
+\authors  "J.R.R. Tolkien, impl. David Fraser based on work of Talagan (Benjamin Babut)"
+
+\world      primary_related_to_arda
+\invention  jrrt
+
+\charset  cirth_ds true
+
+\if dots_for_spaces
+  \outspace CIRTH_PUNCT_MID_DOT
+\else
+  \** We redefine the output space to have something beautiful, especially with erebor1 and erebor2 **\ 
+  \outspace CIRTH_SPACE_BIG
+\endif
```

Comment:
Move this logic to the post processor (new feature).

Reply:
Done...